test/test_linearizer.py

```diff
@@ -1036,14 +1036,16 @@ def helper_arg_acc_dtype(c: Tensor, expected_dtype:DType):
   @unittest.skipUnless(Device[Device.DEFAULT].renderer.tensor_cores, "test requires tensor cores")
   def test_tensor_cores(self):
     for tc in Device[Device.DEFAULT].renderer.tensor_cores:
-      if (getenv("EMULATE_CUDA") or getenv("EMULATE_INTEL")) and (tc.dtype_in == dtypes.bfloat16 or tc.dtype_out == dtypes.bfloat16): continue
```

Comment:
simplified this logic (don't check for tc.dtype_out == dtypes.bfloat16) as there is no accumulation with bf16 without bf16 as input

Reply:
nah i would keep it as is. it's better to explicit check conditions than simplify based on assumptions that might change later

Reply:
Sure, adding that check back!